config/kernel/linux-rockchip64-edge.config

```diff
@@ -1009,6 +1009,9 @@ CONFIG_ATH10K_SDIO=m
 CONFIG_ATH10K_USB=m
 CONFIG_ATH11K=m
 CONFIG_ATH11K_PCI=m
+CONFIG_ATH12K=m
+CONFIG_ATH12K_DEBUG=y
+CONFIG_ATH12K_TRACING=y
```

Comment:
I don't think we need to enable DEBUG and TRACKING as it will be lots of things in the logs - which are not meant for end users.